presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/main/grouphome/GroupHomeScreen.kt

```diff
@@ -149,3 +180,226 @@ private fun GroupTag(
         )
     }
 }
+
+@Composable
+private fun GroupFloatingButton(
+    modifier: Modifier,
+    onClickGroupMake: () -> Unit,
+    onClickGroupEnter: () -> Unit,
+) {
+    var isExpanded by remember { mutableStateOf(false) }
+
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.End,
+    ) {
+        if (isExpanded) {
+            FloatingButtonContent(
+                modifier = Modifier
+                    .background(
+                        color = Color.White,
+                        shape = RoundedCornerShape(10.dp),
```

Comment:
```suggestion
                        shape = RoundedCornerShape(16.dp),
```
엇 이거 16.dp 인 것 같은데 혹시 내가 다른걸로 잘못 착각하고 본건가..?!

Reply:
아냐아냐... 내가 잘못봤네..? 늘 고마워 계원아 진쫘. 디테일한 리뷰 지렸다.